build.gradle

```diff
@@ -99,19 +99,19 @@ configurations.all {
 
 dependencies {
     // TODO: Unpack the "starter" and include just what we use
-    compile 'org.springframework.boot:spring-boot-starter-web'
-    compile 'org.springframework:spring-jdbc'
-    compile 'io.springfox:springfox-swagger2:2.7.0'
+    compile 'ch.qos.logback:logback-classic:1.2.3'       // Logger
+    compile 'com.google.cloud:google-cloud-bigquery:1.66.0'
+    compile 'com.google.cloud:google-cloud-storage:1.66.0'
     compile 'io.springfox:springfox-swagger-ui:2.7.0'
+    compile 'io.springfox:springfox-swagger2:2.7.0'
+    compile 'org.apache.commons:commons-dbcp2:2.5.0'     // For database connection support
     compile 'org.apache.commons:commons-lang3:3.8.1'
     compile 'org.apache.directory.studio:org.apache.commons.io:2.4'
-    compile 'org.apache.commons:commons-dbcp2:2.5.0'     // For database connection support
+    compile 'org.liquibase:liquibase-core:3.6.3'         // For upgrade
     compile 'org.postgresql:postgresql:42.2.5'           // Postgres jdbc driver
-    compile 'com.google.cloud:google-cloud-bigquery:1.66.0'
-    compile 'com.google.cloud:google-cloud-storage:1.66.0'
-    compile 'org.liquibase:liquibase-core:3.6.1'         // For upgrade
     compile 'org.slf4j:slf4j-api:1.7.25'                 // Logging facade
-    compile 'ch.qos.logback:logback-classic:1.2.3'       // Logger
+    compile "org.springframework.boot:spring-boot-starter-web:${springBootVersion}"
+    compile 'org.springframework:spring-jdbc:5.1.6.RELEASE'
```

Comment:
I sorted these alphabetically -- the real changes are from setting versions on spring-jdbc and spring-boot-starter-web + bumping liquibase to 3.6.3.